src/wp-includes/class-wp-scripts.php

```diff
@@ -143,13 +143,13 @@ class WP_Scripts extends WP_Dependencies {
 	private $dependents_map = array();
 
 	/**
-	 * Holds a reference to the allowed script loading strategies.
+	 * Holds a reference to the delayed (non-blocking) script loading strategies.
 	 * Used by methods that validate loading strategies.
 	 *
 	 * @since 6.3.0
 	 * @var string[]
 	 */
-	private $allowed_strategies = array( 'blocking', 'defer', 'async' );
+	private $delayed_strategies = array( 'defer', 'async' );
```

Comment:
@westonruter (cc @joemcgill )this is more of a thought (and a nitpick one at that) but when thinking along the lines of future proofing this implementation, we've had discussions around the potential of new script loading strategies being introduced to the HTML spec in the future. This may or may never happen, we don't know.

What if said new strategies were not _delayed_ strategies but something entirely different? I honestly can not think of an example, but I would not discount the fact that it's possible. Let's take the experimental `blocking` one as an example, what if there became an inherent need to use that in the future, that would not be a delayed strategy, but would equally be an important one to (re) introduce. 

Are we being _too_ limiting in naming conventions by specifically calling this `$delayed_strategies ` as opposed to the former `$allowed_strategies`? I'm playing devils advocate here, but this body of work is to enhanced the WP_Scripts API with a loading strategy, not but not necessarily limited to just a _delayed_ loading strategy.

Perhaps I'm over thinking this for now given that the entire body of work _is_ centred around delayed loading strategies only at present 😄

Reply:
Yeah, I thought about that, but given these members are all `private` we are free to do that refactor in the future if needed. In the mean time, we know the absence of a strategy means blocking for back-compat, so we can always rely on that.

Reply:
Sounds good to me @westonruter